code/modules/reagents/reagent_containers/food/drinks/juicebox.dm

```diff
@@ -1,11 +1,58 @@
 /obj/item/chems/food/drinks/juicebox
+	icon = 'icons/obj/juicebox.dmi'
+	icon_state = "juicebox_base"
+	blend_mode = BLEND_OVERLAY
```

Comment:
I believe BLEND_OVERLAY is the default blending mode for objects - any specific reason to set it again?